lib/std/os.zig

```diff
@@ -4354,6 +4354,35 @@ pub fn fork() ForkError!pid_t {
     }
 }
 
+pub fn getPageSize() usize {
```

Comment:
Should this be `getMinimumPageSize`?
Many architectures have multiple page sizes

Reply:
getDefaultPageSize it is.